src/basilcc/serialize.lzz

```diff
@@ -313,4 +328,6 @@ namespace
 #include <limits>
 #include <numeric>
 #include <stdexcept>
+//#define SerializeWriter BinaryWriter
+#define SerializeWriter TextWriter

```

Comment:
I see, so this is where you can switch to text. Is the idea to have a TextReader to deserialize the data?

Ideally you want to load the parser tables (the fsm) as fast as possible, so a binary format is used. But you are right that the binary format as it is here is not portable. So right now on a different architecture you'd have compile the lzz grammar (with basilcc) then use it with lzz. Really just need a binary format that *is* portable. Then you'd still have the speed.